coalib/bearlib/aspects/decorators.py

```diff
@@ -41,7 +50,25 @@ def _new_func(self, *args, **kwargs):
                             kwargs[arg] = aspect_instance.tastes[
                                 aspect_value.name]
 
-            return func(self, *args, **kwargs)
+            profile_output = kwargs.get('profile_output', False)
+            profile_commands = kwargs.get('profile_commands',False)
+            flag = 0
+            if str(profile_output) is not u'False':
+                flag = 1
+                profile_bears = profile_output
+                kwargs.pop('profile_output')
+            elif str(profile_commands) is not u'False' :
+                flag = 1
+                profile_bears = profile_commands
+                kwargs.pop('profile_commands')
+
+            if flag:
+                str_profile_bears = str(profile_bears).lower().strip()
+                return profile_bears_decorator(func)(self,*args, **kwargs) if not (
+                    str_profile_bears == u'false') else func(self,*args, **kwargs)
```

Comment:
Line is longer than allowed. (82 > 80)

*Origin: LineLengthBear, Section: `linelength`.*